composer.json

```diff
@@ -13,6 +13,7 @@
 		"files": [
 			"inc/healthcheck/namespace.php",
 			"inc/healthcheck/cavalcade/namespace.php",
+			"inc/environment_indicator/namespace.php",
```

Comment:
@roborourke I moved loading into composer and run `composer dump-autoload` and the changes don't appear.
I've also tried to remove symlink to the package and re-install it again, still no bingo :(

What am I missing here?

Reply:
You appear to have removed the bit that called `Environment_Indicator\bootstrap()` in the main `inc/namespace.php` file.

Reply:
Classic :) thanks!